### PR TITLE
cmd/tailscaled: rename variables to be more descriptive

### PR DESCRIPTION
**Pull request by maisem**
renamed from `useNetstack` to `onlyNetstack` which is 1 letter more but
more descriptive because we always have netstack enabled and `useNetstack`
doesn't convey what it is supposed to be used for. e.g. we always use
netstack for Tailscale SSH.

Also renamed shouldWrapNetstack to handleSubnetsInNetstack as it was only used
to configure subnet routing via netstack.

Updates tailscale/corp#8020

Signed-off-by: Maisem Ali <maisem@tailscale.com>

### CODE REVIEW

File: cmd/tailscaled/tailscaled.go [hunk 1]
```diff
@@ -540,24 +540,24 @@ func getLocalBackend(ctx context.Context, logf logger.Logf, logid string) (_ *ip
...
 		}
 		logf("wgengine.NewUserspaceEngine(tun %q) error: %v", name, err)
 		errs = append(errs, err)
 	}
 	return nil, false, multierr.New(errs...)
 }
 
-func shouldWrapNetstack() bool {
+func handleSubnetsInNetstack() bool {
 	if v, ok := envknob.LookupBool("TS_DEBUG_WRAP_NETSTACK"); ok {
```
Comment (bradfitz) [inline cmd/tailscaled/tailscaled.go:561]: this env too then?

File: cmd/tailscaled/tailscaled.go [hunk 2]
```diff
@@ -540,24 +540,24 @@ func getLocalBackend(ctx context.Context, logf logger.Logf, logid string) (_ *ip
...
+			return e, userspaceNetworking, nil
 		}
 		logf("wgengine.NewUserspaceEngine(tun %q) error: %v", name, err)
 		errs = append(errs, err)
 	}
 	return nil, false, multierr.New(errs...)
 }
 
-func shouldWrapNetstack() bool {
+func handleSubnetsInNetstack() bool {
```
Comment (bradfitz) [inline cmd/tailscaled/tailscaled.go:560]: docs then while you're here and apparently caring about such details?

**Review by bradfitz — APPROVED** [review]
Not sure this is more than a neutral change (longer words but not much more descriptive/disambiguating?) but sure, if you care strongly?

File: cmd/tailscaled/tailscaled.go (hunk 1 above)
Comment (maisem) [reply cmd/tailscaled/tailscaled.go]: Done

File: cmd/tailscaled/tailscaled.go (hunk 2 above)
Comment (maisem) [reply cmd/tailscaled/tailscaled.go:567]: Done.

**Comment by maisem** [comment]
> Not sure this is more than a neutral change (longer words but not much more descriptive/disambiguating?) but sure, if you care strongly?

Renamed to `onlyNetstack`, I think that is at least more descriptive in comparison to `useNetstack` because `use` doesn't clarify use for what. `only` at least implies that we only have `netstack` and nothing else (i.e. no tun).

**Pull request merged**